packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -111,6 +112,11 @@ export const routeComponent = (Wrapped, isPage, locals) => {
             return component.shouldGetProps ? component.shouldGetProps(args) : defaultImpl()
         }
 
+        static getLayout(page) {
+            console.log('route-component getLayout defaultLayout')
+            return getLayout(page)
```

Comment:
This `getLayout` is imported from somewhere in the react-sdk package. Does react-sdk need to provide some default layouts? I thought layouts would live inside templates only.

Reply:
Correct. This was only an example of defining and using a default layout outside a template. 
But you're right. The Layouts will probably live along with the rest of the components we extract from the template in a separate package.

I removed this default layout from the SDK for now since creating new packages is being addressed in separated spikes.